packages/react/src/components/table/utils/table-utils.ts

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 
 export function calculateStickyColumns(
     stickyColumns: boolean[],
-    headerCells: HTMLCollectionOf<HTMLTableHeaderCellElement>,
-    rows: HTMLCollectionOf<HTMLTableRowElement>,
+    headerCells: NodeListOf<HTMLTableHeaderCellElement>,
```

Comment:
`HTMLTableHeaderCellElement` seems to be deprecated.
```suggestion
    headerCells: NodeListOf<HTMLTableCellElement>,
```